public/locales/zh/common.json

```diff
@@ -0,0 +1,34 @@
+{
+    "User Profiles": "用户配置文件",
+    "Profile": "配置文件",
+    "Add New Server": "添加新服务器",
+    "Fancy Mumble Title": "Fancy Mumble",
+    "Unknown User": "未知用户",
+    "Search": "搜索",
+    "Search Channel": "搜索频道",
+    "Search Tenor": "在 Tenor 搜索 GIF",
+    "Open In Browser": "在浏览器打开",
+    "Muted": "已静音",
+    "Deafened": "已关闭声音",
+    "Joined": "已加入",
+    "User ID": "用户 ID",
+    "Settings": "设置",
+    "None": "无",
+    "Image too large": "[[ 图片太大了捏 ( {{size}} 最大体积是 {{maximum}}) ]]",
+    "write something": "发点什么吧 :D",
+    "Feature Not Implemented": "{{feature}} (Not Implemented)",
```

Comment:
I use this string to mark that something is not implemented, e.g. "Autopilot (Not Implemented)"